ludwig/features/numerical_feature.py

```diff
@@ -207,11 +207,14 @@ def __init__(self, feature, encoder_obj=None):
     def forward(self, inputs):
         assert isinstance(inputs, torch.Tensor)
         assert inputs.dtype == torch.float32 or inputs.dtype == torch.float64
-        assert len(inputs.shape) == 1 or (len(inputs.shape) == 2 and inputs.shape[1] == 1)
+        assert len(inputs.shape) == 1 or (
+                    len(inputs.shape) == 2 and inputs.shape[1] == 1)
```

Comment:
Here wondering if it would be better to be more prescriptive and support only one or the other clauses of the or. Allthough there's a worls where in the future we do more feature augmentation and `inputs.shape[1]` could be > 1, so maybe keeping the flexibility is ok. @tgaddair opinions?

Reply:
@w4nderlust I believe removing the assert for `len(inputs.shape) == 2 and inputs.shape[1] == 1` (i.e. supporting for a tensor of shape `batch` without support shape `batch x 1`) breaks some tests in `test_model_save_and_load.py`. Can't remember which ones right now, but I'd added the second assert since I figured the two shapes were functionally the same, and supporting `batch x 1` seemed more consistent with the other feature types.